src/sql/src/plan/statement/ddl.rs

```diff
@@ -1820,18 +1639,15 @@ pub(crate) fn load_generator_ast_to_generator(
     };
 
     let mut available_subsources = BTreeMap::new();
-    for (i, (name, desc)) in load_generator.views().iter().enumerate() {
+    for (name, desc) in load_generator.views().into_iter() {
```

Comment:
just a nit that the `for x in xs` pattern implicitly calls `into_iter()` on `xs`

```suggestion
    for (name, desc) in load_generator.views() {
```